configs/HGLRCF722AIOV2/config.h

```diff
@@ -0,0 +1,124 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                   STM32F7X2
+#define BOARD_NAME                      HGLRCF722AIOV2
+#define MANUFACTURER_ID                 HGLR
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_MAX7456
+
+#define BEEPER_PIN                      PC13
+#define MOTOR1_PIN                      PB4
+#define MOTOR2_PIN                      PB5
+#define MOTOR3_PIN                      PB0
+#define MOTOR4_PIN                      PB1
+#define MOTOR5_PIN                      PA15
+#define MOTOR6_PIN                      PB3
+#define MOTOR7_PIN                      PB6
+#define MOTOR8_PIN                      PB7
+#define RX_PPM_PIN                      PA3
+#define RX_PWM1_PIN                     PA2
+#define RX_PWM2_PIN                     PA1
+#define RX_PWM3_PIN                     PA0
+#define LED_STRIP_PIN                   PA08
+#define UART1_TX_PIN                    PA09
+#define UART2_TX_PIN                    PA02
+#define UART3_TX_PIN                    PB10
+#define UART4_TX_PIN                    PA0
+#define UART5_TX_PIN                    PC12
+#define UART6_TX_PIN                    PC06
```

Comment:
```suggestion
#define UART6_TX_PIN                    PC6
```

Reply:
Thanks. Long time ago I used this script